.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: tests
+on: push
+jobs:
+  test:
+    name: Test (Ruby ${{ matrix.ruby }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ]
```

Comment:
Is it worth having this in the matrix with only one entry? Perhaps `runs-on: ubuntu-latest` may be less maintenance?

Reply:
Good idea. I'm all for less maintenance.

Reply:
Updated in e160298b8bae0828feab24079b79faadcb131b56